process.go

```diff
@@ -71,6 +75,10 @@ func (p *process) Wait(ctx context.Context) (uint32, error) {
 	if err != nil {
 		return UnknownExitStatus, err
 	}
+	// first check if the task has exited
+	if status, _ := p.Status(ctx); status == Stopped {
+		return UnknownExitStatus, errdefs.ErrUnavailable
```

Comment:
Hm, ok. I found that `ExitStatus` is not part of `Process` now...
I suggest that `Wait` should return exit status without error, so that we could completely avoid the race.

However, that requires we add exit status into `Process`.